service/gcsutils/remotefs/types.go

```diff
@@ -22,7 +22,7 @@ type FileInfo struct {
 	NameVar    string
 	SizeVar    int64
 	ModeVar    os.FileMode
-	ModTimeVar time.Time
+	ModTimeVar int64 // Serialization of time.Time breaks in travis, so use an int
```

Comment:
Yeah, it breaks in travis, but not on my local machine. It has something to do with the locale/location. not sure what